fix hardcoding of compiler command & flags, by passing down $CXX, $CXX and $CXXFLAGS
--- Makefile.orig	2014-02-21 01:20:50.000000000 +0100
+++ Makefile	2019-03-14 16:20:18.415277482 +0100
@@ -4,6 +4,6 @@
 
 
 all depend test clean:
-	set -e ; for file in $(SUBDIRS) ; do make -C $$file $(MAKEFLAGS:%=-%) $@ ; done
+	set -e ; for file in $(SUBDIRS) ; do make -C $$file $@ CC="${CC}" CXX="${CXX}" FLAGS="${CFLAGS}"; done
 
 
